src/analytics/README.md

```diff
@@ -1,7 +1,9 @@
 clone project from GitHub.
+
```

Comment:
Hi @Mnoble-19 , thanks for this. 
- We could aim to avoid any type of changes to the Analytics microservice.
- On that note, please remove these "new line" changes accordingly just to ensure that this PR is just about changes for Insights microservice and nothing else. Thanks!